src/thor/triplegbuilder.cc

```diff
@@ -120,6 +139,12 @@ void UpdateIncident(const std::shared_ptr<const valhalla::IncidentsTile>& incide
     const auto& meta = valhalla::baldr::getIncidentMetadata(incidents_tile, *incident_location);
     *new_incident->mutable_metadata() = meta;
 
+    // Set iso country code (2 & 3 char codes) on the new incident obj created for this leg
+    std::string country_code_iso_2 = country_code_from_edge(tile, de);
+    std::string country_code_iso_3 = valhalla::tyr::get_iso_3166_1_alpha3(country_code_iso_2);
```

Comment:
we shouldnt be calling tyr from thor. this makes me think actually that this util funciton should be moved to like `baldr` or `midgard` or something